api/controllers/record-utils.js

```diff
@@ -217,25 +217,21 @@ const createByForm = (data, { locale }={}) => {
     throw new PublicError('Missing required field: message');
   }
 
-  const fields = ['from', 'message', 'reported_date', 'locale', 'gateway_ref'];
```

Comment:
This old code was quite misleading - these field definitions and the subsequent call to `_.pick()` are irrelevant to the final data that ends up in the db.

Reply:
Wowzer that is misleading..

Reply:
I think this was just an oversight in the de-kanso PR.